destination/config_test.go

```diff
@@ -85,3 +86,24 @@ func TestConfig_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestConfig_ParseTopic_ErrorOnIndexNotFound(t *testing.T) {
```

Comment:
I'd rename the test, since IndexNotFound doesn't tell me much. Maybe something like "template returns empty string" or something along those lines.